pkg/cmd/psa/psa.go

```diff
@@ -82,10 +83,32 @@ func (o *PSAOptions) Run() error {
 // checkNamespacePodSecurity collects the pod security violations for a given
 // namespace on a stricter pod security enforcement.
 func (o *PSAOptions) checkNamespacePodSecurity(ns *corev1.Namespace) (*PodSecurityViolation, error) {
-	nsCopy := ns.DeepCopy()
+	labels := getLabels(ns)
+	targetLevel := o.level
+
+	if targetLevel == "" {
+		strictestLabels := getStrictestLabels(labels)
+		if strictestLabels.Has(psapi.EnforceLevelLabel) {
+			// We don't need to check the namespace if logging levels are
```

Comment:
why "logging levels"? aren't you checking for "enfoce levels"?

Reply:
If the logging levels (audit and warn) are the same as enforce levels, we don't need to check those namespaces. Nothing should be able to run that would violate them.

Reply:
logging levels is an ambiguous term. Please mention audit and warn explicitly.